presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java

```diff
@@ -261,6 +262,22 @@ private boolean queryCreated(DispatchQuery dispatchQuery)
                 .collect(toImmutableList());
     }
 
+    @Managed
+    public long getQueuedQueryCount()
+    {
+        return queryTracker.getAllQueries().stream()
+                .filter(query -> query.getBasicQueryInfo().getState() == QUEUED)
+                .count();
+    }
+
+    @Managed
+    public long getRunningQueryCount()
+    {
+        return queryTracker.getAllQueries().stream()
+                .filter(query -> query.getBasicQueryInfo().getState() != QUEUED && !query.getBasicQueryInfo().getState().isDone())
```

Comment:
This doesn't match the semantics of `RunningQueries` from QueryManagerStats - that class counted a query as running only when transitioning to `RUNNING`.  But that counted queries in all previous states as `QueuedQueries`, which didn't seem right.